README.md

```diff
@@ -30,11 +30,13 @@ It currently exposes 'pagination', 'pageSize' and 'table'.
 ```
 ``` javascript
 actions: {
-    getData(queryObj){
+    getData(queryObj, defaultSerializedObject){
       return this.get('store').query('data', queryObj);
     }
   }
 ```
+The defaultSerializedObject is a object serialize for C# API that will be able to deserialize a array object for the filters. The queryObj is provider if you want to format it differently.
```

Comment:
+The defaultSerializedObject is an object serialized for the C# API so that it can deserialize an array object for the filters (queryParams). The queryObj is provided if you want to serialize it differently. You can use the queryObj straight up if your API accepts JSON in queryParams.